sw/device/lib/dif/dif_rv_core_ibex.h

```diff
@@ -76,6 +76,36 @@ typedef struct dif_rv_core_ibex_addr_translation_region {
   dif_rv_core_ibex_addr_translation_pair_t dbus;
 } dif_rv_core_ibex_addr_translation_region_t;
 
+/**
+ * Ibex error status detected by `rv_core_ibex` peripheral.
+ */
+typedef enum dif_rv_core_ibex_error_status {
+  /**
+   * Register transmission integrity error.
+   */
+  kDifRvCoreIbexErrorStatusRegisterTransmissionIntegrity = 1 << 0,
```

Comment:
I'd take a look at the CSRNG DIF for the preferred way to do this, rather than matching the hardware's indices exactly.

Reply:
I'm going to leave this way with the `static_assert` suggested by @moidx below.